model/PermissionsService.php

```diff
@@ -119,6 +127,13 @@ public function savePermissions(
         foreach ($actions as $processedResource) {
             $processedResource();
         }
+
+        $this->eventManager->trigger(
+            new DacAffectedUsersEvent(
```

Comment:
Not only users. Also roles provided to this event. But `DacAffectedUsersAndRolesEvent` too complicated to read